keyboards/kiwikey/kawii9/rev2/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
for your consideration: this setting can be moved to features, in info.json, which would result in this line to be removed.
```suggestion
```